vibrato/src/tokenizer.rs

```diff
@@ -1,11 +1,12 @@
 //! Viterbi-based tokenizer.
 pub(crate) mod lattice;
+pub mod worker;
 
 use crate::dictionary::Dictionary;
 use crate::errors::{Result, VibratoError};
 use crate::sentence::Sentence;
-use crate::worker::Worker;
 use lattice::Lattice;
+use worker::Worker;
```

Comment:
I think that the absolute path `crate::tokenizer...` is better for consistency.

Reply:
I fixed all parts.
https://github.com/daac-tools/vibrato/pull/23/commits/82ff1b38d43a7d852f8844da080c77f7f23ce0d3